docs/reference/client-worker-configuration.md

```diff
@@ -50,4 +49,50 @@ The file may be called by any name but should have the following structure:
 | `componentDelivery` | Optional | Specifies the delivery settings for the worker's components. Analogous to [`component_delivery` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#component-delivery). |
 | `componentSettings` | Optional | Defines component specific settings. Analogous to [`component_settings` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#component-settings). |
 | `permissions` | Optional | Specifies the worker's entity permissions. Analogous to [`permissions` in the previous configuration version](https://docs.improbable.io/reference/latest/shared/worker-configuration/permissions). |
-| `uploadConfiguration` | Optional | Specifies the client worker's build directory for each platform. If provided, the `spatial alpha cloud upload` CLI command uploads the given client worker, enabling you to use it with the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher). <br><br> It contains three optional fields (`windows`, `macos`, `linux`) which each contain one required `localBuildDirectory` string field. The `localBuildDirectory` field specifies the directory containing the worker binary to be executed. <br><br> **Note:** Your built-out worker must be self-contained in the `localBuildDirectory` and all files that your worker might depend on while running must be in this folder. Files outside this folder will not be uploaded to the cloud as part of your project assembly. |
+| `uploadConfiguration` | Optional | Describes how to start the worker. If provided, the `spatial alpha cloud upload` CLI command uploads the given client worker, enabling you to use it with the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher). See below for more information. |
```

Comment:
```suggestion
| `uploadConfiguration` | Optional | Describes how to start the worker. If provided, the `spatial alpha cloud upload` CLI command uploads the given client-worker, enabling you to use it with the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher). See below for more information. |
```

I was going to add "instance" or "type" to the places where you've used "worker", but I don't know what's correct for "uploads the given client-worker..." - instance, or type? Or do neither of these quite work?

Reply:
It would in this case be "zipped client-worker executable". I'll change the sentence.